tools/walletextension/wallet_extension.go

```diff
@@ -89,6 +91,9 @@ func (we *WalletExtension) Shutdown() {
 	}
 }
 
+// Used to check whether the server is ready.
+func (we *WalletExtension) handleReady(http.ResponseWriter, *http.Request) {}
```

Comment:
I introduced this new method to check readiness. It used to hit the regular `/` endpoint to check for liveness, but that made the wallet extension actually try and process the body, and led to harmless but misleading errors.